samples/caf_sensor_manager/sysbuild/nrf54h20pdk_nrf54h20_cpuppr.conf

```diff
@@ -0,0 +1 @@
+SB_CONFIG_REMOTE_BOARD="nrf54h20pdk_nrf54h20_cpuppr"
```

Comment:
note, for reference here: these "sysbuild files", exist temporarily because of a quoting bug, see https://github.com/zephyrproject-rtos/zephyr/issues/69552

Reply:
I used these to have remote present in certain build types only.